src/ui/core/zif_abapgit_gui_event.intf.abap

```diff
@@ -19,4 +19,4 @@ INTERFACE zif_abapgit_gui_event
     RAISING
       zcx_abapgit_exception.
 
-ENDINTERFACE.
+ENDINTERFACE.
```

Comment:
will the missing newline cause a diff?

Reply:
I wondered the same thing, but importing seems okay,
![image](https://user-images.githubusercontent.com/5888506/107606316-ff773100-6c35-11eb-904b-27d58bec7c6e.png)